pkg/cmd/pulumi/org_search.go

```diff
@@ -170,7 +200,11 @@ func newSearchCmd() *cobra.Command {
 	)
 	cmd.PersistentFlags().VarP(
 		&scmd.outputFormat, "output", "o",
-		"Output format. Supported formats are 'table', 'json', and 'yaml'.",
+		"Output format. Supported formats are 'table', 'json', 'csv', and 'yaml'.",
+	)
+	cmd.PersistentFlags().Var(
```

Comment:
Should there be a default? Like a `,` (comma)?

Reply:
There is - when one is not provided, comma is used by default.